### PR TITLE
Implemented ModMessagePopup

### PR DESCRIPTION
**Pull request by TAImatem**

### CODE REVIEW

**Comment by amazingalek** [comment]
very cool! looking forward to testing it

File: OWML.ModHelper.Menus/ModMessagePopup.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            newPopup.Initialize(newPopupObject.GetComponent<PopupMenu>());
+            return newPopup;
+        }
+
+        public void ShowMessage(string message, bool addCancel = false, string okMessage = "OK", string cancelMessage = "Cancel")
+        {
+            if (_twoButtonPopup == null || IsOpen)
+            {
+                Console.WriteLine("Failed to create popup for a following message:");
+                Console.WriteLine(message);
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModMessagePopup.cs:49]: Wrong console

File: OWML.ModHelper.Menus/ModMessagePopup.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            IsOpen = true;
+            _twoButtonPopup.OnPopupConfirm += OnPopupConfirm;
+            _twoButtonPopup.OnPopupCancel += OnPopupCancel;
+            _twoButtonPopup.EnableMenu(true);
+            _twoButtonPopup.SetUpPopup(message, InputLibrary.confirm, addCancel ? InputLibrary.cancel : null,
+                new ScreenPrompt(InputLibrary.confirm, okMessage), new ScreenPrompt(InputLibrary.cancel, cancelMessage), true, addCancel);
+            _twoButtonPopup.GetValue<Text>("_labelText").text = message;
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModMessagePopup.cs:57]: extract screen prompts to variables, will increase readability

**Comment by amazingalek** [comment]
and merge from dev

File: OWML.ModHelper.Menus/ModMessagePopup.cs (hunk 1 above)
Comment (TAImatem) [reply OWML.ModHelper.Menus/ModMessagePopup.cs]: OK

File: OWML.ModHelper.Menus/ModMessagePopup.cs (hunk 2 above)
Comment (TAImatem) [reply OWML.ModHelper.Menus/ModMessagePopup.cs]: OK

**Pull request merged**